### PR TITLE
fix: style Get Directions link as link

### PR DESCRIPTION
**Pull request by uroslates**
Closes https://jira.tools.sap/browse/CXSPA-8814

### CODE REVIEW

File: feature-libs/storefinder/components/store-finder-list-item/store-finder-list-item.component.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
       href="{{ getDirections(location) }}"
       target="_blank"
       rel="noopener noreferrer"
-      class="btn btn-sm btn-secondary btn-block cx-button"
```
Comment (Pio-Bar) [inline feature-libs/storefinder/components/store-finder-list-item/store-finder-list-item.component.html:53]: A small change, but I think it's still considered breaking. Let's use a feature flag here.

**Review by Pio-Bar — CHANGES_REQUESTED** [review]

File: feature-libs/storefinder/components/store-finder-list-item/store-finder-list-item.component.html (hunk 1 above)
Comment (uroslates) [reply feature-libs/storefinder/components/store-finder-list-item/store-finder-list-item.component.html:53]: Should be addressed. Added new feature flag for these kind of changes.

File: projects/core/src/features-config/feature-toggles/config/feature-toggles.ts [hunk 2]
```diff
@@ -797,6 +804,7 @@ export const defaultFeatureToggles: Required<FeatureTogglesInterface> = {
   a11yFacetKeyboardNavigation: true,
   a11yUnitsListKeyboardControls: true,
   a11yCartItemsLinksStyles: true,
+  a11yStyleExternalLinksAsLinks: true,
```
Comment (Pio-Bar) [inline projects/core/src/features-config/feature-toggles/config/feature-toggles.ts]: New feature flags need to be turned off by default for the first 6 months.

**Review by Pio-Bar — CHANGES_REQUESTED** [review]

File: projects/core/src/features-config/feature-toggles/config/feature-toggles.ts (hunk 2 above)
Comment (uroslates) [reply projects/core/src/features-config/feature-toggles/config/feature-toggles.ts]: OK thanks for the info. Should be updated now!

**Review by Pio-Bar — APPROVED** [review]

**Review by Zeyber — APPROVED** [review]

**Review by developpeurweb — APPROVED** [review]

**Pull request merged**